modules/books/client/createViewEditBooks/editBook/components/book-details-edit.js

```diff
@@ -0,0 +1,39 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('books.component')
+    .component('bookDetailsEdit', {
+      bindings: {
+        book: '=',
+        edit: '='
+      },
+      controller: BookDetailsEditCtrl,
+      controllerAs: 'vm',
+      templateUrl: 'modules/books/client/createViewEditBooks/editBook/components/book-details-edit.html'
+
+    });
+
+  BookDetailsEditCtrl.inject = [];
+
+  function BookDetailsEditCtrl() {
+
+    this.save = function(book) {
```

Comment:
let be consistent here as well and make this" var vm = this;" and bind stuff to vm